Content/Items/Misc/Accessories.Corpseflower.cs

```diff
@@ -1,10 +1,15 @@
-﻿using StarlightRiver.Content.Items.BaseTypes;
-using System.Collections.Generic;
+﻿using Mono.Cecil.Cil;
+using MonoMod.Cil;
+using StarlightRiver.Content.Items.BaseTypes;
+using StarlightRiver.Core.Systems.InstancedBuffSystem;
+using StarlightRiver.Helpers;
+using System;
 
 namespace StarlightRiver.Content.Items.Misc
 {
 	public class Corpseflower : CursedAccessory
 	{
+		public int[] maxTimeLefts = new int[100];
```

Comment:
Is there something along the lines of Main.MaxCombatTexts we can use here instead of a magic number? Also needs a linebreak after this line